### PR TITLE
Add Feature: ignore specific tmux buffers

### PR DESCRIPTION
**Pull request by kevintraver**
PR for #51

### CODE REVIEW

**Comment by aserowy** [comment]
ty for the pr. I have commented couple small things.

Kind regards
Alexander

**Comment by kevintraver** [comment]
Hmmm, Im not sure I see the comments

**Review by aserowy — CHANGES_REQUESTED** [review]
my fault, i didnt finished the review :D

File: lua/tmux/configuration/options.lua [hunk 1]
```diff
@@ -35,6 +35,9 @@ local M = {
         -- overwriting the unnamed register. Thus, ddp would not be possible.
         sync_deletes = true,
 
+        -- ignore specific tmux buffers
+        ignore_buffers = { "tmp" },
```
[the remark this answers is not in the document]
Comment (kevintraver) [reply lua/tmux/configuration/options.lua:39]: Ok makes sense

**Comment by aserowy** [comment]
yeah, i will look into it tomorrow. No idea why its not copying :D

**Comment by aserowy** [comment]
I fixed it, BUT the version does not function... vim.tbl_deep_extend has problems with this kind of object structure. :dagger:

I guess we must use an unordered table like

```lua
{
  "buffer_name" = true,
}
```

I have to take a closer look into this one.

**Comment by aserowy** [comment]
Ok, im unsure if the problem was really vim.tbl_deep_extend or my own fat fingers :dagger: 

I reworked the structure to an unordered list and now it works like a charm!

**Review by aserowy — APPROVED** [review]

**Comment by aserowy** [comment]
ty for your pr @kevintraver

**Comment by aserowy** [comment]
closes #51

**Pull request merged**